app/main/views/g_cloud.py

```diff
@@ -121,7 +121,7 @@ def get_service_by_id(service_id):
             service_data,
             content_loader.get_builder(framework_slug, 'display_service').filter(
                 service_data
-            ),
+            ).summary(service_data),
```

Comment:
I wonder if it would be preferable to have this call inside the Service [presenter] constructor?